centreon/packages/ui/src/ActionsList/index.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+import { makeStyles } from 'tss-react/mui';
+import { useTranslation } from 'react-i18next';
+
+import { SvgIconProps } from '@mui/material';
+import MenuList from '@mui/material/MenuList';
```

Comment:
Btw, you can import mui's component from @mui/material directly instead of importing them line by line